### DISCUSSION TITLE
Lospec Palette Importer in Pixelorama

### DISCUSSION THREAD

**Discussion by piotrtrkostrzewski94** [Orama-Interactive/Pixelorama#1005] [Ideas]
Good morning.
Is it possible to add in the future (as an add-on or built-in feature) the option to import palettes from lospec.com (or other sites) directly in Pixelorama?

https://lospec.com/articles/lospec-palette-importer-for-aseprite-now-available

https://sudo-whoami.itch.io/lospec-palette-importer

**Comment by Variable-ind** [comment]
Hy :wave: We did initially plan it, using the Godot Lospec plugin as a reference, but it turned out Lospec has a problem with plugins.
The people at Lospec told the developer of "Godot Lospec Plugin" to remove it, so we won't be able to use it, and if we use something similar they will complain again. https://twitter.com/hiulit/status/1585643492507672578

**Reply by OverloadedOrama** [reply]
I asked at the Lospec Discord server yesterday why they are okay with the new Aseprite importer, but not with the Godot Lospec Plugin, and they replied that the Godot Lospec Plugin wasn't using the Lospec API and it was replacing their browser function, taking away page views. And they told me that they are okay if such plugins exist but they use their API, so we should be able to make such an importer without having issues with Lospec, as long as we use their API.

**Reply by Variable-ind** [reply]
Interesting, that's good news then 🙂

**Reply by piotrtrkostrzewski94** [reply]
So there's a chance :) Thank you very much :) . Have a nice day :)

**Comment by OverloadedOrama** [comment]
There you go, an extension for Pixelorama v1.0 that downloads and imports Lospec palettes! https://github.com/OverloadedOrama/Lospec-Palette-Importer

**Reply by piotrtrkostrzewski94** [reply]
Thank you very much :) Have a nice weekend :)
